src/assets/i18n/de.json

```diff
@@ -0,0 +1,237 @@
+    "NOT_AVAILABLE_WATCH_ONLY": "Nicht verfügbar für reine Leseadresse",
+    "SETTINGS": "Einstellungen",
+    "VERSION": "Version",
+    "WALLET_BACKUP": "Wallet Backup"
+  },
+  "SHOW_ADVANCED": "Zeige Fortgeschrittene",
+  "SKIP": "Überspringen",
+  "STATUS": "Status",
+  "TRANSACTIONS_PAGE": {
+    "2ND_SIGNATURE_CREATION": "Zweite Signatur erstellen",
+    "ADD_ADDRESS_TO_CONTACTS": "Füge {{ address }} zu Kontakten hinzu",
+    "AMOUNT": "Menge",
+    "CONFIRMATIONS": "Bestätigungen",
+    "CONTACT_NAME": "Neuer Kontakt Name (optional)",
+    "COPY_TXID": "Kopiere TXID",
+    "CREATE_TRANSACTION_ERROR": "Konnte Transaktion nicht erstellen, checke die Internetverbindung",
+    "DELEGATE_REGISTRATION": "Delegate Registration",
+    "ENCRYPT_SAVE_SECOND_PASSPHRASE": "Möchtest du deine zweite Passphrase speichern und verschlüsseln?",
+    "FEE": "Gebühr",
+    "FETCHING_TRANSACTIONS": "Lade Transaktionen",
+    "FROM": "Von",
+    "ID": "Id",
+    "INVALID_ADDRESS_ERROR": "Ungültige Adresse",
+    "INVALID_CONTACT_ERROR": "Ungültiger Kontakt",
+    "INVALID_FORM_ERROR": "Ungültige Daten",
+    "MULTISIGNATURE_CREATION": "Multisignatur erzeugen",
+    "OPEN_EXPLORER": "Öffnen in Explorer",
+    "QRCODE_COPY": "Klicke auf die Adresse, um sie zu kopieren oder klicke auf das Icon um den QR Code zu teilen.",
+    "RECEIVE": "Erhalten",
+    "RECEIVED": "Erhalten",
+    "RECEIVED_FROM": "Erhalten von",
+    "RECIPIENT": "Empfänger",
+    "SECOND_SIGNATURE_CREATION": "Zweite Signatur erzeugen",
+    "SECOND_PASSPHRASE_NOT_ENTERED": "Die zweite Signatur wurde nicht eingegeben",
+    "SEND": "Senden",
+    "SEND_ALL": "Alle senden",
+    "SEND_TOKEN_TO_ADDRESS": "Sende {{ token }} an {{ address }}",
+    "SENT": "Gesendet",
+    "SENT_TO": "Gesendet an",
+    "SEND_TRANSACTION": "Sende Transaktion",
+    "SENT_SUCCESS": "Transaktion erfolgreich gesendet!",
+    "SMART_BRIDGE": "SmartBridge (optional)",
+    "TO": "An",
+    "TRANSACTION": "Transaktion"
+  },
+  "TYPE": "Typ",
+  "WALLETS_PAGE": {
+    "ADD_WALLET_ERROR": "Konnte Wallet nicht hinzufügen",
+    "ADD_WALLET_TEXT": "Um Wallet zu importieren oder hinzuzufügen, klicke den nachfolgenden Button",
+    "ADDRESS": "Adresse",
+    "ALERT_SUCCESSFULLY_ENCRYPTED_SECOND_PASSPHRASE": "Die zweite Passphrase wurde erfolgreich verschlüsselt",
+    "BACKUP_ACCOUNT": "WICHTIG! Sichere diese geheime Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen und ALLE DEINE ARK MÜNZEN WERDEN VERLOREN SEIN.",
+    "BACKUP_PASSPHRASE": "Sichere diese geheime Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen.",
+    "BACKUP_SECOND_PASSPHRASE": "Sichere diese zweite Passphrase, ohne diese Wörter kannst du dein Wallet nicht wiederherstellen.",
+    "BACKUP_SECOND_PASSPHRASE_SECURELY_RETYPE_TEXT": "Du kannst ein Passwort deiner Wahl hinzufügen. Vergiss nicht dies zu sichern, du benötigst es für die Bestätigung.",
```

Comment:
I think `password` is wrong and should be `passphrase` (also in english version).

`Du kannst ein Passphrase deiner Wahl hinzufügen. Vergiss nicht diesen zu sichern, du musst ihn nachher nochmals zur Bestätigung eingeben.`